addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -80,7 +81,8 @@ class AccountMove(models.Model):
     @api.depends('move_type', 'company_id')
     def _compute_l10n_es_tbai_is_required(self):
         for move in self:
-            move.l10n_es_tbai_is_required = move.is_sale_document() \
+            move.l10n_es_tbai_is_required = (move.is_sale_document() or move.is_purchase_document() and move.company_id.l10n_es_tbai_tax_agency == 'bizkaia'
```

Comment:
so vouchers (in_receipts, out_receipts) are not to be included? It doesn't seem very consistent: previously we were checking if move_type.startswith('xxx'), or move_type in (...)

Probably better to choose one and stick to it for the whole module, imo